youtube_dl/extractor/pornhub.py

```diff
@@ -265,8 +265,16 @@ def _real_extract(self, url):
 
         self._login(host)
 
+        # The age verification cookie may have a different name based on the location.
+        # Setting multple cookies doesn't seem to be big deal so we prefere that solution instead
+        # of guessing the right cookie out of an IP we don't currently have.
         self._set_cookie(host, 'age_verified', '1')
 
+        if 'thumbzilla.com' in host:
```

Comment:
Let's try the version from https://github.com/yt-dlp/yt-dlp/issues/4299#issuecomment-1489631938.

Reply:
I let my comment about why we set multiple cookies. ^^
Feel free to edit it.